Source/WebCore/Modules/webxr/XRCompositionLayer.idl

```diff
@@ -31,6 +31,7 @@
     Exposed=Window,
     JSGenerateToJSObject,
     JSGenerateToNativeObject,
+    IgnoreSubclassesWhenGeneratingToJSObject
```

Comment:
Not sure I understand when we should do `SuppressToJSObject` and when we instead should do IgnoreSubclassesWhenGeneratingToJSObject. The change log doesn’t mention how you decided this.